pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -183,3 +184,64 @@ def test_dataframe_constructor_with_different_dtype_raises():
     xpr = "Cannot coerce extension array to dtype 'int64'. "
     with tm.assert_raises_regex(ValueError, xpr):
         pd.DataFrame({"A": arr}, dtype='int64')
+
+
+class TestOps(BaseDecimal, base.BaseOpsTests):
+    def check_op(self, s, op_name, other):
+
+        short_opname = op_name.strip('_')
+        if short_opname[0] == 'r':
+            short_opname = short_opname[1:]
+        op = getattr(operator, short_opname)
+        result = op(s, other)
+        expected = s.combine(other, op)
+        self.assert_series_equal(result, expected)
+
+    def test_arith_scalar(self, data, all_arithmetic_operators):
+        # scalar
+        op_name = all_arithmetic_operators
+        s = pd.Series(data)
+        self.check_op(s, op_name, decimal.Decimal(1.5))
```

Comment:
if you update the scalar in the base test, I think this one can be inherited?

Reply:
Yes, that was possible